packages/node/src/NodeFileSystem.ts

```diff
@@ -19,16 +19,61 @@ export class NodeFileSystem implements FileSystem {
    * Create new NodeFileSystem class instance.
    *
    * @param baseDataPath The base path to use for reading and writing data files used within the framework.
-   * Files will be created under baseDataPath/.afj directory. If not specified, it will be set to homedir()
+   * Files will be created under baseDataPath/.credo directory. If not specified, it will be set to homedir()
    * @param baseCachePath The base path to use for reading and writing cache files used within the framework.
-   * Files will be created under baseCachePath/.afj directory. If not specified, it will be set to homedir()
+   * Files will be created under baseCachePath/.credo directory. If not specified, it will be set to homedir()
    * @param baseTempPath The base path to use for reading and writing temporary files within the framework.
-   * Files will be created under baseTempPath/.afj directory. If not specified, it will be set to tmpdir()
+   * Files will be created under baseTempPath/.credo directory. If not specified, it will be set to tmpdir()
    */
   public constructor(options?: { baseDataPath?: string; baseCachePath?: string; baseTempPath?: string }) {
-    this.dataPath = options?.baseDataPath ? `${options?.baseDataPath}/.afj` : `${homedir()}/.afj/data`
-    this.cachePath = options?.baseCachePath ? `${options?.baseCachePath}/.afj` : `${homedir()}/.afj/cache`
-    this.tempPath = `${options?.baseTempPath ?? tmpdir()}/.afj`
+    this.dataPath = options?.baseDataPath ? `${options?.baseDataPath}/.credo` : `${homedir()}/.credo/data`
+    this.cachePath = options?.baseCachePath ? `${options?.baseCachePath}/.credo` : `${homedir()}/.credo/cache`
+    this.tempPath = `${options?.baseTempPath ?? tmpdir()}/.credo`
+  }
+
+  /**
+   * Migrate data from .afj to .credo if .afj exists.
+   * Copy the contents from old directory (.afj) to new directory (.credo).
+   */
+  public async migrateWalletToCredoFolder(walletId: string) {
```

Comment:
I think the file system should stay very generic. So instead of adding this method `migrateWalletToCredoFolder` in the file system (which then needs to be implemented by every file system), I'd move it to the migration method. This can then call the needed methods `exists` and `copyDirectory` (i think this method can also copy files now? Should we create one generic `copy` method that can do both directories (using `recursive` property) and files?)